charts/plugin-br-pix-direct-jd/templates/plugin-br-pix-direct-jd-job/configmap.yaml

```
@@ -8,6 +8,8 @@ data:
   # -- Default Environment variables for the plugin-br-pix-direct-jd
   NODE_ENV: {{ .Values.job.configmap.NODE_ENV | default "development" | quote }}
   DEBUG: {{ .Values.job.configmap.DEBUG | default "false" | quote }}
+  LOG_LEVEL: {{ .Values.job.configmap.LOG_LEVEL | default "info" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)